src/Common/Uuid/UuidRegistry.php

```diff
@@ -183,4 +183,33 @@ public static function uuidToBytes($uuidString)
     {
         return Uuid::fromString($uuidString)->getBytes();
     }
+
+    /**
+     * Check if UUID String is Valid
+     * @return boolean
+     */
+    public static function isValidStringUUID($uuidString)
+    {
+        return (UUID::isValid($uuidString));
+    }
+
+    /**
+     * Check if UUID String is Empty
+     * @return boolean
+     */
+    public static function isEmptyStringUUID($uuidString)
+    {
+        return (!empty($uuidString) &&
+            $uuidString != null &&
+            $uuidString != '');
+    }
+
+    /**
+     * Check if UUID Brinary is Empty
+     * @return boolean
+     */
+    public static function isEmptyBinaryUUID($uuidString)
+    {
+        return ($uuidString != '\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0');
```

Comment:
for this, note basically includes what is empty in the string in addition to this crazy binary, so something like:
```
return ((self::isEmptyStringUUID($uuidString)) && ($uuidString != '\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0'))
```